docs/sdk/rust/contract-structure/near-bindgen.md

```diff
@@ -94,3 +94,7 @@ pub fn my_method(&mut self) {
 ...
 }
 ```
+
+## Interaction with other macros
```

Comment:
This could be better as a detail. Thought I don't understand in which context a smart-contract developer would find this error.